doc/languages-frameworks/python.section.md

```diff
@@ -833,6 +833,7 @@ used in `buildPythonPackage`.
 - `pythonRemoveBinBytecode` to remove bytecode from the `/bin` folder.
 - `setuptoolsBuildHook` to build a wheel using `setuptools`.
 - `setuptoolsCheckHook` to run tests with `python setup.py test`.
+- `virtualenvShellHook` to source a `virtualenv` at the `venvDir` location. A `virtualenv` is created if it does not yet exist.
```

Comment:
Typo in the hook name, should be `venvShellHook`.
Further I think we should stick to using the term `venv` here to avoid ambiguity (at least if we don't support both) - while having the same goals `venv` and `virtualenv` are two separate things with possibly slightly different behaviors.